README.rst

```diff
@@ -37,16 +37,16 @@ Davon sind auch technische Pflichtfelder, namentlich `_typ` und `_version` nicht
 Ob sie anzugeben sind, entscheidet die Anwendung.
 BO4E gibt sinnvolle Datenstrukturen vor, wie diese in der Praxis genutzt werden können und wollen wir aber nicht vorschreiben.
 
-Verknüpfungen zwischen Objekten
+Verweise zwischen Objekten
 -------------------------------
 Viele Objekte verweisen aufeinander und sind miteinander verknüpft.
 Dabei sind unsere Designentscheidungen:
 
-* Alle Verknüpfungen sind optional.
-* Verknüpfungen sind, wo sie naheliegend sind, im BO4E Standard vordefiniert (z.B. 1 Messlokation hat n Zähler).
-* Aber wir haben nicht jede theoretisch denkbare Verknüpfungen implementiert (z.B. allein dass User Zähler unter Angabe einer Email-Adresse suchen können, heißt nicht, dass der Zähler eine optionale Eigenschaft `emailAdresse` haben muss.)
-* Generell sollen zwischen zwei BOs bi-direktional sein, zwischen BOs und COMs aber nur unidirektional (z.B. soll jeder Zähler wissen zu welcher Messlokation er gehört aber eine Adresse muss nicht wissen, welchem Geschäftspartner, welcher Messlokation oder welcher Rechnung sie zugeordnet ist).
-* Zwischen COMs sollen keine Verknüpfungen bestehen (d.h. es sollen keine COMs in COMs eingebettet werden).
+* Alle Verweise sind optional.
+* Verweise sind, wo sie naheliegend sind, im BO4E Standard vordefiniert (z.B. 1 Messlokation hat n Zähler).
+* Aber wir haben nicht jede theoretisch denkbare Verweise implementiert (z.B. allein dass User Zähler unter Angabe einer Email-Adresse suchen können, heißt nicht, dass der Zähler eine optionale Eigenschaft `emailAdresse` haben muss.)
+* Generell sollen Verweise zwischen zwei BOs bi-direktional sein, zwischen BOs und COMs aber nur unidirektional (z.B. soll jeder Zähler wissen zu welcher Messlokation er gehört aber eine Adresse muss nicht wissen, welchem Geschäftspartner, welcher Messlokation oder welcher Rechnung sie zugeordnet ist).
+* Zwischen COMs sollen keine Verweise bestehen (d.h. COMs aus verschiedenen BOs sollen nicht aufeinander verweisen).
```

Comment:
also soll es keine "indirekten" brücken bzw tunnel über die COMs geben? den satz in klammern finde ich nicht intuitiv klar.

Reply:
Hm okay, dann müssen wir noch Formulierungen brainstormen. 
Also COMs dürfen COMs beinhalten, aber die Verweise sollen nicht dafür genutzt werden zwei COMs aus verschiedenen BOs zu verknüpfen, so wie ich es verstanden habe.

Reply:
```suggestion
* COMs können zwar weitere COMs beinhalten, jedoch sollte dies nicht dafür genutzt werden von einem COM eines BOs auf das COM eines anderen BOs zu verweisen, sondern immer als ganzes Objekt darzustellen. 
Bsp.: Die Adresse in Ansprechpartner ist identisch zur Adresse zur Lokationsadresse in der Marktlokation. Dann sollen beide Adressen als vollständiges Object dargestellt werden, statt nur einen Verweis von einer Adresse auf die andere.
```

Reply:
Wie wäre diese Formulierung?

Reply:
Finde ich schon sehr gut, nur den Teilsatz "sondern immer als ganzes Objekt darzustellen." müsste man noch anpassen: 
COMs sind dann immer als ganzes Objekt darzustellen? Vielleicht?

Reply:
Regel: wir können aus einem BO oder einem COM auf ein anderes BO verweisen.

Reply:
![grafik](https://github.com/bo4e/BO4E-python/assets/73471037/b6a89f55-68a7-449a-be06-bd6a852b08c6)